### DISCUSSION TITLE
Model is not appearing in AR mode. Means, when I click on view in space only camera is turned on but the model is not loaded/ augment.

### DISCUSSION THREAD

**Discussion by saurabhburadkar** [google/model-viewer#2714] [Q&A]
The model is not appearing in AR mode. When I click on view in space, the camera is turned on, but the model is not loaded/ augment.

**Comment by saurabhburadkar** [comment]
DEV I am completely available for you If you want any type of logs or anything. I am using POCO m2 pro and the browser is chrome.

**Comment by elalish** [comment]
We'll need a lot more detail to be able to help. Please post a link where we can repro. What device/browser/OS do you have and what are the versions? If you can  share a screen recording of the problem that would help a lot too.

**Comment by saurabhburadkar** [comment]
I used a model viewer astronaut example. My device name is POCO M2 pro. The browser is chrome and the OS is Android 10 - MIUI 12.0.4. 
Video link:- https://cdn.glitch.com/33f32773-c5cd-4f21-81a9-23de1a23eebd%2Fa.mp4?v=1629907632858
I also used this before and they worked too. But now they are not working. I don't know why? But I am completely available to help you.

**Reply by elalish** [reply]
That is strange indeed; can you paste in the contents of your console? WebXR generally has helpful error messages.

**Reply by saurabhburadkar** [reply]
![image](https://user-images.githubusercontent.com/71985034/130838333-f70bec82-7f9e-4d58-913f-5761df9fb8eb.png)
This is surprised me too.

**Reply by elalish** [reply]
Yeah, dev tools doesn't have access to the camera feed, so that's normal. I'm surprised there's no console warnings though. @klausw Do you have any idea what's going on? Any hope of a repro device over there somewhere?

**Reply by saurabhburadkar** [reply]
Can you explain to me what is repro device is so I can help you?

**Reply by elalish** [reply]
I just mean we need a device that shows the same behavior (reproduces your results). For instance it works fine on my Pixel 3, so I can't repro.

**Reply by klausw** [reply]
It's a bit surprising that the "place model" UI wasn't showing. Is it possible that the model got placed at a bad location, i.e. straight up or down or behind the viewer?

Also, could you try some basic WebXR AR samples to see if WebXR is working outside of model-viewer?

https://immersive-web.github.io/webxr-samples/immersive-ar-session.html
https://immersive-web.github.io/webxr-samples/hit-test.html

Thanks!

**Reply by saurabhburadkar** [reply]
https://user-images.githubusercontent.com/71985034/130842895-d2af4978-860b-4ef1-a4b0-e78016d51ed1.mp4

The video.

**Comment by klausw** [comment]
Hm, that's odd. One more would be https://immersive-web.github.io/webxr-samples/ar-barebones.html - this doesn't use the reported poses at all, and should just draw a colored rectangle in the middle of the screen.

**Reply by saurabhburadkar** [reply]
https://user-images.githubusercontent.com/71985034/130844274-acd5284a-83db-49c2-ae57-ff36e96dc595.mp4

**Reply by klausw** [reply]
Thanks! This looks as if the WebGL content layer either isn't visible in the composited output, or isn't being updated correctly.

**Reply by saurabhburadkar** [reply]
any time,
So what do I have to do to solve this issue?

**Comment by klausw** [comment]
It's hard to say what would need to be done to fix this. Could you please provide the full Chrome version number from chrome://version, for example "92.0.4515.159 (Official Build) (64-bit)" for my desktop browser?

Also, would you be willing to try one of the newer Chrome builds, for example https://play.google.com/store/apps/details?id=com.chrome.canary or https://play.google.com/store/apps/details?id=com.chrome.dev ? No need to take video of these if they have the same behavior.

**Reply by saurabhburadkar** [reply]
POCO M2 Pro chrome version "92.0.4515.159" and my desktop browser "92.0.4515.159".
Models did not appear on camera. Same behavior.

**Reply by saurabhburadkar** [reply]
which hardware required to work augmented reality in device? I think its my device issue. Becuase its suddenly stop working. I also restart my device, but no luck. Is there any application that test that AR is working or not?

**Reply by saurabhburadkar** [reply]
Today I received a notification from Google that your backup is stopped so log in to your google account. After I log in to the account I tested the model viewer and its worked. Now I am surprised how this worked? 

https://user-images.githubusercontent.com/71985034/130914299-58af63b5-a25e-439a-a92b-c3e819b44873.mp4

**Reply by elalish** [reply]
Now  that is truly baffling! My only guess is maybe the play store stopped downloading updates when your account info was out of date and somehow you got stuck  with incompatible versions of Chrome and ARCore? Wild guess... Glad it's working again!

**Reply by saurabhburadkar** [reply]
This is nothing short of a miracle :) but, thank you very much dev's for your all help.
